dev-libs/granite/granite-5.5.0.ebuild

```diff
@@ -0,0 +1,78 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+VALA_MIN_API_VERSION=0.40
+BUILD_DIR="${WORKDIR}/${P}-build"
+
+inherit meson vala xdg
+
+DESCRIPTION="Elementary OS library that extends GTK+"
+HOMEPAGE="https://github.com/elementary/granite"
+SRC_URI="https://github.com/elementary/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc +introspection"
```

Comment:
I don't understand what this USE flag does. How does granite change depending on the `deps[introspection]` state?

Reply:
@juippis well... `option('introspection', type: 'boolean', value: true, description: 'Whether to build introspection files')`

Reply:
But you don't have it defined in `src_configure` :)

Reply:
@juippis Blame the last maintainer. >:-)

Reply:
@juippis Oh! Optional introspection support is only on master!